website/docs/topics/feature-flags/best-practices-using-feature-flags-at-scale.md

```diff
@@ -0,0 +1,351 @@
+
+To keep complexity down, we generally advise not to have complex targeting rules for both the parent and the child flag simultaneously. This can make it harder to achieve consistency for end users. It also increases the complexity for the user of the feature flag system. For example, if both the parent and the child are configured to a 50% rollout, are you then exposing the feature to 50% or 25% of the user base? The answer is that it depends on the identifier used to roll the flag out.
+
+### 12. Don’t use flags to manage configuration.
+
+Both feature flags and configuration settings control an application’s behavior. However, it's crucial to distinguish between feature flagging systems and configuration management systems and use each for its intended purpose.  Otherwise, you risk management nightmares and security vulnerabilities
+
+Feature flags are intended for dynamic and temporary use, giving you [runtime control](feature-flag-best-practices.md#1-enable-run-time-control-control-flags-dynamically-not-using-config-files) to enable or disable functionality in a live environment. This flexibility supports [short-lived experiments](feature-flag-best-practices.md#7-make-feature-flags-short-lived-do-not-confuse-flags-with-application-configuration), such as A/B testing or gradual rollouts, instead of relying on configuration settings that require an application redeploy to take effect. Despite the ease of turning off and on features or functionality with feature flags, they should not be repurposed for managing configuration.
+
+A dedicated config management system should manage long-term configuration settings. These systems are designed to handle sensitive information such as API keys and access credentials securely, unlike feature flag systems, which typically do not encrypt flag states. Misusing feature flags for configuration can lead to security risks and management complexities.
+
+A good rule of thumb is that if the data is static (you don’t expect it to change without restarting your application), needs encryption, or contains PII (Personal Identifiable Information), it’s probably better suited for your Configuration management system than a feature flag system.
+
+
+<table>
+  <tr>
+   <td>Configuration system
+   </td>
+   <td>Feature flag system
+   </td>
+  </tr>
+  <tr>
+   <td>Is: Long-lived, static during runtime
+   </td>
+   <td>Is: Short-lived, changes during runtime
+   </td>
+  </tr>
+  <tr>
+   <td>Examples of things to update with the config system: 
+<ul>
+
+<li>Database or server credentials
+
+<li>Server port
+
+<li>CORS headers
+
+<li>API base URL
+
+<li>SSO setup
+</li>
+</ul>
+   </td>
+   <td>Examples of things to update with feature flag system:
+<ul>
+
+<li>Dark mode/light mode
+
+<li>Stable / Beta feature
+
+<li>New or old algorithm
+
+<li>A/B test
+</li>
+</ul>
+   </td>
+  </tr>
+</table>
```

Comment:
The formatting here is off and it's breaking the build. Replacing with a new version.

```suggestion
<table>
  <tr>
    <th>Configuration system</th>
    <th>Feature flag system</th>
  </tr>
  <tr>
    <td>Is: Long-lived, static during runtime</td>
    <td>Is: Short-lived, changes during runtime</td>
  </tr>
  <tr>
    <td>
      Examples of things to update with the config system:
      <ul>
        <li>Database or server credentials</li>
        <li>Server port</li>
        <li>CORS headers</li>
        <li>API base URL</li>
        <li>SSO setup</li>
      </ul>
    </td>
    <td>
      Examples of things to update with feature flag system:
      <ul>
        <li>Dark mode / light mode</li>
        <li>Stable/Beta feature</li>
        <li>New or old algorithm</li>
        <li>A/B test</li>
      </ul>
    </td>
  </tr>
</table>
```